diff --git a/node_modules/expo-camera/ios/Current/CameraView.swift b/node_modules/expo-camera/ios/Current/CameraView.swift
index b2e7f9e..e14d3e3 100644
--- a/node_modules/expo-camera/ios/Current/CameraView.swift
+++ b/node_modules/expo-camera/ios/Current/CameraView.swift
@@ -140,9 +140,14 @@ public class CameraView: ExpoView, EXAppLifecycleListener, EXCameraInterface, Ca
   let onAvailableLensesChanged = EventDispatcher()
 
   internal var deviceOrientation: UIInterfaceOrientation {
-    UIApplication.shared.connectedScenes.compactMap {
-      $0 as? UIWindowScene
-    }.first?.interfaceOrientation ?? .unknown
+    if #available(iOS 13.0, *) {
+      return UIApplication.shared.connectedScenes.compactMap {
+        $0 as? UIWindowScene
+      }.first?.interfaceOrientation ?? .unknown
+    } else {
+      // Fallback for iOS < 13.0
+      return UIApplication.shared.statusBarOrientation
+    }
   }
 
   required init(appContext: AppContext? = nil) {
